test/kitchen/test/integration/win-sysprobe-test/rspec/sysprobe_spec_helper.rb

```diff
@@ -0,0 +1,52 @@
+require "rspec/core/formatters/base_text_formatter"
+
+class CustomFormatter
+  RSpec::Core::Formatters.register self, :example_passed, :example_failed, :dump_summary, :dump_failures, :example_group_started, :example_group_finished
+
+  def initialize(output)
+    @output = output
+  end
+
+  # Remove "."'s from the test execution output
+  def example_passed(_)
+  end
+
+  # Remove "F"'s from the test execution output
+  def example_failed(_)
+  end
```

Comment:
Are these intended to be blank? The comments above seem to be indicating otherwise

Reply:
By being blank, they prevent those characters from being output.